### PR TITLE
 document stub usage in vs-code (pybricks/support#937)

### PR DESCRIPTION
**Pull request by kai-morich**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -12,6 +12,11 @@ used to generate the `official documentation`_.
 See the `contributor's guide <CONTRIBUTING.md>`_ for acceptable changes and
 instructions to build the documentation locally.
 
+You can use the API stubs in this repository for syntax highlighting when
+programming the EV3 with VS code. To enable syntax highlighting, remove the 
+``"python.languageServer"="None"`` line in the ``.vscode/settings.json`` file
+generated by the *Lego Mindstromes EV3 MicroPython* extension.
```
Comment (dlech) [inline README.rst:18]: Let's use the official name.

```suggestion
generated by the *LEGO® MINDSTORMS® EV3 MicroPython* extension.
```

Comment (dlech) [reply README.rst:18]: We could also make this a link.

File: README.rst [hunk 2]
```diff
@@ -12,6 +12,11 @@ used to generate the `official documentation`_.
 See the `contributor's guide <CONTRIBUTING.md>`_ for acceptable changes and
 instructions to build the documentation locally.
 
+You can use the API stubs in this repository for syntax highlighting when
```
Comment (dlech) [inline README.rst:15]: The subs don't provide syntax highlighting, they provide code completion (intellesense).

**Review by dlech — COMMENTED** [review]
Thanks! I've made some suggestions to improve this.

File: README.rst (hunk 2 above)
Comment (kai-morich) [reply README.rst:15]: provides syntax highlighting and code completion.
w/o stubs unknown classes are shown in white, with stubs in green and also offer code completion. We could mention both.

Comment (kai-morich) [reply README.rst:15]: For me syntax highlighting, code completion and (life) type checking somehow belong together, so I did not mention in detail.

Comment (dlech) [reply README.rst:15]: Sure we can mention both.

File: README.rst [hunk 3]
```diff
@@ -12,6 +12,11 @@ used to generate the `official documentation`_.
 See the `contributor's guide <CONTRIBUTING.md>`_ for acceptable changes and
 instructions to build the documentation locally.
 
+You can use the API stubs in this repository for syntax highlighting and code 
+completion when programming the EV3 with VS code. To enable, remove the
```
Comment (dlech) [inline README.rst:16]:
```suggestion
completion when programming the EV3 with VS Code. To enable, remove the 
```

**Comment by dlech** [comment]
Thanks!

**Pull request merged**